### PR TITLE
unit tests for tripledb is added

### PR DESCRIPTION
**Pull request by AbhijitVyas**

### CODE REVIEW

File: src/db/tripledb.plt [hunk 1]
```diff
@@ -96,5 +96,10 @@ test('tripledb tell triple with time scope extension'):-
     tripledb_tell(test_datatype:'Lecturer4', test_datatype:'last_name', 'Spiendler', T_S1),
     assert_true(tripledb_ask(test_datatype:'Lecturer4', test_datatype:'last_name', 'Spiendler', T_S1, _, _)).
 
+% test for unit
+test('tripledb tell triple with Unit'):-
```
Comment (daniel86) [inline src/db/tripledb.plt:100]: could you add *assert_false* case first?

File: src/db/tripledb.plt [hunk 2]
```diff
@@ -96,5 +96,10 @@ test('tripledb tell triple with time scope extension'):-
...
+    tripledb_tell(test_datatype:'Lecturer4', test_datatype:'height', unit(double(2.1),'meter')),
```
Comment (daniel86) [inline src/db/tripledb.plt]: you can also wrap this call in *assert_true*. Then you get more informative message in case the test fails.

**Review by daniel86 — CHANGES_REQUESTED** [review]

File: src/db/tripledb.plt (hunk 1 above)
Comment (AbhijitVyas) [reply src/db/tripledb.plt:100]: done.

File: src/db/tripledb.plt (hunk 2 above)
Comment (AbhijitVyas) [reply src/db/tripledb.plt]: done.

**Review by daniel86 — APPROVED** [review]

**Pull request merged**